requirements.txt

```diff
@@ -0,0 +1,7 @@
+bitarray==2.7.3
+commoncode==31.0.2
+click==8.1.3
+simplejson==3.18.4
+unicodecsv==0.14.1
+pytest==7.2.2
```

Comment:
Similarly as above, pytest and pytest-xdist should only be present in the requirements-dev.txt, and not here.

Reply:
Done.